packages/discovery/src/discovery/output/saveDiscoveryResult.ts

```diff
@@ -27,6 +28,51 @@ export interface SaveDiscoveryResultOptions {
   flatSourcesFolder?: string
   discoveryFilename?: string
   metaFilename?: string
+  skipHints?: boolean
+}
+
+export async function saveConfigHints(
+  rootPath: string,
+  results: Analysis[],
+  options: SaveDiscoveryResultOptions,
+): Promise<void> {
+  const filePath = path.join(rootPath, 'config.hints.jsonc')
+  await rimraf(filePath)
```

Comment:
do want to remove it when we're not going to overwrite it?

Reply:
Yes, if there are no more hints then we want the hints from the previous run to disappear - probably someone has just added the hints to the config file.